tests/Processor/Braintree/Query/SubscriptionQueryTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace TeamGantt\Dues\Tests\Processor\Braintree\Query\Subscription;
+
+trait SubscriptionQueryTest
+{
+    public function testSubscriptionSearchWithDaysPastDue()
+    {
+        $query = $this->dues->makeSubscriptionQuery();
+        $overdue = $query->daysPastDue('>=', 1)->fetch();
+
+        // this assumes that you have at least one past due subscription in your processor.
+        // for the case of braintree, you can create a past due subscription by using an
+        // invalid payment method `fake-invalid-nonce` and setting a start date in the future.
+        // The subscription will fail to start, and immediately go into an past due status.
+        $this->greaterThan(0, $overdue);
```

Comment:
Can we just have the test create the overdue subscription as described here?

Also, what is `$this->greaterThan`? Does that actually make an assertion?

Reply:
I missed the description in the PR - please disregard the test comment 🙇🏻

Reply:
:+1:  > intellisense was yelling at me original for assertGreaterThan and was recommending that... it's now not yelling at me. back to `assertGreaterThan()`